### ISSUE TITLE
Include Microsoft.EntityFrameworkCore.InMemory on the procedure

### ISSUE THREAD

**Issue by thiagofborn** [dotnet/AspNetCore.Docs#15929]
Dear  blogger builders, 

I think you should include the installation of "Microsoft.EntityFrameworkCore.InMemory"  in the project. Otherwise, on a fresh VS 2019 installation, the "UseInMemoryDatabase" will be not available.  So, add an extra step of the package installation before the step "Update Startup.cs." I could be via "NuGet Package Manager Console" for the sake of simplicity as follows: 

Install-Package Microsoft.EntityFrameworkCore.InMemory

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 583ae3e7-5915-f22b-e54c-6c100cf2a7da
* Version Independent ID: 95f2549e-29f9-e31a-b4d0-432b971643ff
* Content: [Tutorial: Create a web API with ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-3.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/first-web-api.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/first-web-api.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
It's in the instructions.

**Comment by DisneyUK** [comment]
> Dear blogger builders,
> 
> I think you should include the installation of "Microsoft.EntityFrameworkCore.InMemory" in the project. Otherwise, on a fresh VS 2019 installation, the "UseInMemoryDatabase" will be not available. So, add an extra step of the package installation before the step "Update Startup.cs." I could be via "NuGet Package Manager Console" for the sake of simplicity as follows:
> 
> Install-Package Microsoft.EntityFrameworkCore.InMemory
> 
> #### Document Details
> ⚠ _Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking._
> 
> * ID: 583ae3e7-5915-f22b-e54c-6c100cf2a7da
> * Version Independent ID: 95f2549e-29f9-e31a-b4d0-432b971643ff
> * Content: [Tutorial: Create a web API with ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-3.0&tabs=visual-studio)
> * Content Source: [aspnetcore/tutorials/first-web-api.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/first-web-api.md)
> * Product: **aspnet-core**
> * Technology: **aspnetcore-tutorials**
> * GitHub Login: @Rick-Anderson
> * Microsoft Alias: **riande**

**Comment by DisneyUK** [comment]
Please review again and amend, I followed the project instructions to the letter and double checked and had exactly the problem thiagofborn described ("UseInMemoryDatabase" not available) on a new VS 2019 install. As thiag suggested I ran "Install-Package Microsoft.EntityFrameworkCore.InMemory" from the package manager console which fixed it

**Comment by Rick-Anderson** [comment]
![image](https://user-images.githubusercontent.com/3605364/69920963-ed92df00-1431-11ea-8829-43c0159a7922.png)

**Comment by DisneyUK** [comment]
it's badly written "preceding instructions" makes no sense, suggest rewording

**Comment by dukedvl** [comment]
I agree, it's poorly written. I missed this step as well until coming down to leave my own complaint.. Spacing the bulletin 1 line lower/ emboldening the InMemory package in the same style as the SqlServer package would help. (It's a second step, it gets glossed over when it's lumped under the first step)

**Comment by colinmxs** [comment]
Our interns also missed this step in the instructions. Please emphasize.

**Comment by Rick-Anderson** [comment]
@wadepickett this has been fixed and is unrelated to #19859
